modules/ding_debt_easy/includes/ding_debt_easy.admin.inc

```diff
@@ -0,0 +1,250 @@
+    'test' => 'https://test.api.dibspayment.eu/',
+  ]);
+
+  $form['ding_debt_easy']['endpoints']['ding_debt_easy_api_endpoints'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Nets Easy payments API'),
+    '#descriptions' => t('These endpoints should not change, but if required they can be overridden in settings.php.'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#tree' => TRUE,
+  ];
+
+  $form['ding_debt_easy']['endpoints']['ding_debt_easy_api_endpoints']['type'] = [
+    '#type' => 'checkbox',
+    '#title' => t('Use testing endpoint'),
+    '#default_value' => $default_api['type'],
+  ];
+
+  $form['ding_debt_easy']['endpoints']['ding_debt_easy_api_endpoints']['prod'] = [
+    '#type' => 'textfield',
+    '#title' => t('Production payments endpoint'),
+    '#description' => '',
+    '#default_value' => $default_api['prod'],
+    '#disabled' => TRUE,
+    '#required' => TRUE,
+  ];
+
+  $form['ding_debt_easy']['endpoints']['ding_debt_easy_api_endpoints']['test'] = [
+    '#type' => 'textfield',
+    '#title' => t('Test payments endpoint'),
+    '#default_value' => $default_api['test'],
+    '#disabled' => TRUE,
+    '#required' => TRUE,
+  ];
+
+  $default_endpoints = variable_get('ding_debt_easy_endpoints', [
+    'callback' => DING_DEBT_EASY_CALLBACK_URL,
+    'cancel' => DING_DEBT_EASY_CANCEL_URL,
+  ]);
+  $form['ding_debt_easy']['endpoints']['ding_debt_easy_endpoints'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Return/callback URL'),
+    '#description' => t('You should not change these URLs as it will/may break payment (mostly here for debug and you really need to known what you are doing to change them.)'),
+    '#collapsible' => TRUE,
+    '#collapsed' => TRUE,
+    '#tree' => TRUE,
+  ];
+
+  $form['ding_debt_easy']['endpoints']['ding_debt_easy_endpoints']['callback'] = [
+    '#type' => 'textfield',
+    '#title' => t('Return for completed payment'),
+    '#default_value' => $default_endpoints['callback'],
+    '#required' => TRUE,
+  ];
+
+  $form['ding_debt_easy']['endpoints']['ding_debt_easy_endpoints']['cancel'] = [
+    '#type' => 'textfield',
+    '#title' => t('Cancel payment callback'),
+    '#default_value' => $default_endpoints['cancel'],
+    '#required' => TRUE,
+  ];
+
+  $form['ding_debt_easy']['ding_debt_easy_config'] = [
+    '#type' => 'fieldset',
+    '#title' => t('Basic configuration'),
+    '#collapsible' => TRUE,
+    '#collapsed' => FALSE,
+    '#tree' => TRUE,
+  ];
+
+  $form['ding_debt_easy']['ding_debt_easy_config']['currency'] = [
+    '#type' => 'select',
+    '#title' => t('Currency'),
+    '#options' => _ding_debt_easy_get_currency_options(),
+    '#default_value' => ding_debt_easy_get_currency(),
+  ];
+
+  $form['ding_debt_easy']['ding_debt_easy_config']['lang'] = [
+    '#type' => 'select',
+    '#title' => t('Language'),
+    '#options' => _ding_debt_easy_get_localization_options(),
+    '#default_value' => ding_debt_easy_get_localization(),
+  ];
```

Comment:
This have been decided by the project group that this should not be dynamic in relation to user langauge.

Reply:
OK I guess. When such decisions which go against the general framework are made it would be good to have them documentation somewhere - either as a code comment or in the commit message.